src/lib/device-online-state.ts

```diff
@@ -176,9 +176,9 @@ export class DeviceOnlineStateManager extends events.EventEmitter {
 
 	private readonly featureIsEnabled: boolean;
 
-	isConsuming: boolean = false;
-	rsmq: RedisSMQ;
-	redis: PromisifedRedisClient;
+	public isConsuming: boolean = false;
+	public rsmq: RedisSMQ;
+	public redis: PromisifedRedisClient;
```

Comment:
I think these were meant to be `private` -- the access modifier was probably forgotten.

Reply:
I see no reason they would need to be `public` 👍